DEVELOPMENT.md

```diff
@@ -83,26 +103,50 @@ To set up locally, clone the repository and navigate to the `frontend` subdirect
     npm install
     ```
 
-4. Build the packages locally.
+4. Build the packages.
 
     ```bash
     npm run build
     ```
 
-    At this point the libraries should be ready to use in any of the projects in the `examples/` directory.
+    OR
 
-5. (Optional) Start the packages in watch mode.
+    Build in watch mode:
 
     ```bash
     npm run dev
     ```
 
-    Alternatively, you can publish the packages locally using Verdaccio:
+At this point the packages should be ready to use in the repository, such as linting/typechecking, IDE support, and the projects in the [`./examples`](./examples) directory.
+
+5. (Optional) Link the `snapwp` command.
+
+    To test the local versions of [`./packages/cli`], you can link it to your global NPM packages:
+
+    ```bash
+    npm link snapwp
+    ```
+
+    For more information see CLI's [Readme](./packages/cli/README.md) file.
+
+6. (Optional) Publish to local Verdaccio registry.
+
+    The following command starts a Verdaccio proxy server on docker and then builds and publishes the packages to the local Verdaccio registry.
 
     ```bash
     npm run publish:local
     ```
 
+    For more information see [@todo]
```

Comment:
What more information should be added here? A link to verdaccio?

Reply:
Yup, if/after the title gets updated, like the other one.